src/service/url-expander/expander.js

```diff
@@ -295,11 +295,11 @@ export class Expander {
   evaluateBinding_(bindingInfo, opt_args) {
     const {encode, name} = bindingInfo;
     let binding;
-    if (hasOwn(bindingInfo, 'prioritized')) {
+    if (bindingInfo.prioritized) {
```

Comment:
looks like these need to check explicitly for `undefined` to handle falsy cases.

Reply:
Interesting. I thought `bingingInfo.prioritized` is always a function. When will it be a false value?

Reply:
macros don't have to be functions. Sometimes they are just static values. Callers can pass in any optional macros as desired e.g. `{FOO: false}`.

Reply:
Thanks. But is there a real use case for that?

Reply:
I think something like this, where they are passing primitives to the expander:
https://github.com/ampproject/amphtml/blob/e92f42d1a4cb7fc76bd0db0e5282e07f7701d097/extensions/amp-a4a/0.1/real-time-config-manager.js#L138-L141

Reply:
I see. Thanks
Any concern if we restrict the macros to resolvers to always be functions?